packages/kbn-unified-data-table/src/utils/get_render_cell_popover.tsx

```diff
@@ -19,7 +19,9 @@ import React, { memo, useEffect } from 'react';
  *
  * */
 export const getCustomCellPopoverRenderer = () => {
-  return memo(function RenderCustomCellPopover(props: EuiDataGridCellPopoverElementProps) {
+  const RenderCustomCellPopoverMemoized = memo(function RenderCustomCellPopoverMemoized(
```

Comment:
```suggestion
  const RenderCustomCellPopoverMemoized = memo(function (
```
Nit: is naming this twice necessary?

Reply:
Just same reason. Unfortunately, const does not give name to a component so we started using below pattern. 

```
const CompName= memo(function CompName() {
 ...
}

```

Reply:
let me know if you any downsides of this pattern.

Reply:
Thanks for explaining! No issues on my end, mainly just curious.